packages/web3torrent/src/library/types.ts

```diff
@@ -121,6 +121,7 @@ export type PeerByTorrent = {
   wire: PaidStreamingWire | PeerWire;
   allowed: boolean;
   funds: string;
+  seederBalance: string;
```

Comment:
What's the difference between funds and seederBalance? Is this your way of having 2 sources of truth?

Reply:
It might seem that way, and it's on my todo list to reduce potential confusion here.

Funds can go up and down. Seeder balance will typically only ever increase. Funds could be renamed to `buffer` or something like that.

Reply:
okay! :-)